rest/route/project.go

```diff
@@ -978,6 +978,63 @@ func (h *getProjectVersionsHandler) Run(ctx context.Context) gimlet.Responder {
 	return resp
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/projects/{project_id}/tasks/{task_id}
+
+type getProjectTasksHandler struct {
+	projectName string
+	taskName    string
+	opts        dbModel.GetProjectTasksOpts
+	url         string
+}
+
+func makeGetProjectTasksHandler(url string) gimlet.RouteHandler {
+	return &getProjectTasksHandler{url: url}
+}
+
+func (h *getProjectTasksHandler) Factory() gimlet.RouteHandler {
+	return &getProjectTasksHandler{url: h.url}
+}
+
+func (h *getProjectTasksHandler) Parse(ctx context.Context, r *http.Request) error {
+	h.projectName = gimlet.GetVars(r)["project_id"]
+	h.taskName = gimlet.GetVars(r)["task_id"]
+	// body is optional
+	b, _ := ioutil.ReadAll(r.Body)
+	if len(b) > 0 {
+		if err := json.Unmarshal(b, &h.opts); err != nil {
+			return errors.Wrap(err, "error parsing request body")
+		}
+	}
+
+	if h.opts.NumVersions == 0 {
+		h.opts.NumVersions = defaultVersionLimit
+	}
+	if h.opts.NumVersions < 1 {
```

Comment:
Preference for this to just error if h.opts.NumVersion < 0, and keep the default handling in GetProjectTasksWithOptions.